microbenchmarks/pom.xml

```diff
@@ -0,0 +1,151 @@
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+    license agreements. See the NOTICE file distributed with this work for additional 
+    information regarding copyright ownership. The ASF licenses this file to 
+    You under the Apache License, Version 2.0 (the "License"); you may not use 
+    this file except in compliance with the License. You may obtain a copy of 
+    the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required 
+    by applicable law or agreed to in writing, software distributed under the 
+    License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS 
+    OF ANY KIND, either express or implied. See the License for the specific 
+    language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.bookkeeper</groupId>
+        <artifactId>bookkeeper</artifactId>
+        <version>4.7.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>microbenchmarks</artifactId>
```

Comment:
There is actually a module `jmh` under `tests`: https://github.com/apache/bookkeeper/tree/master/tests/jmh

We can push this jmh benchmark in that module.

Reply:
I added this one for a few reasons:
- I already had it like that ;)
- jmh-bench did not build even after I fixed version in pom. "ProtocolBenchmark.java:[88,41] constructor AddRequest in class org.apache.bookkeeper.proto.BookieProtocol.AddRequest cannot be applied to given types". 
- I was not sure what the plans fro jmh-bench are, given its current state.

Reply:
don't worry at this patch. I think jmh module was forgot due to some merge issues before. I had the change to fix that. I can incorporate your module in my change.